hail_search/queries/base.py

```diff
@@ -259,7 +259,7 @@ def _load_filtered_table(self, sample_data, intervals=None, annotations=None, an
         parsed_intervals = self._parse_intervals(intervals, **kwargs)
         parsed_annotations = self._parse_annotations(annotations, annotations_secondary, **kwargs)
         self.import_filtered_table(
```

Comment:
possibly out of scope for this PR but I noticed that `import_filtered_table` takes a parameter 'intervals' but I don't see it used anywhere inside that function, so perhaps you could delete it in this PR too